bank.py

```diff
@@ -0,0 +1,166 @@
+import uuid
+from collections import UserDict
+from datetime import datetime
+from functools import wraps
+from typing import Dict, Tuple, Union
+
+from account import Account
+from history_tool import AddHistoryMixIn, History
+
+
+class AccountDoesNotExist(Exception):
+    pass 
+
+
+class AccountExist(Exception):
+    pass
+
+
+class InvalidOpeningBalance(Exception):
+    pass
+
+
+class InsufficientTransferBalance(Exception):
+    pass
+
+
+class DictAccount(UserDict):
+
+    def get(self, key: str):
```

Comment:
get should not raises exception in python way (dict.get return none if key doesnt exist)